### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by vitvadym**
[DEMO LINK](https://vitvadym.github.io/layout_calendar/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="container">
+      <div
+        class="calendar calendar--start-day-wed calendar--month-length-31">
```
Comment (silchyn) [inline src/index.html]:
```suggestion
      <div
        class="calendar calendar--start-day--wed calendar--month-length--31"
      >
```

File: src/index.html [hunk 2]
```diff
@@ -7,6 +7,42 @@
...
+
```
Comment (silchyn) [inline src/index.html]:
```suggestion
```

File: src/styles/block/container.scss [hunk 3]
```diff
@@ -0,0 +1,39 @@
+@import "../utils/_mixins";
+
+.container {
+  @include centered-main-container;
+}
+
+.calendar {
```
Comment (silchyn) [inline src/styles/block/container.scss]: this is not part of `container` block

File: src/styles/block/container.scss [hunk 4]
```diff
@@ -0,0 +1,39 @@
+@import "../utils/_mixins";
```
Comment (silchyn) [inline src/styles/block/container.scss]: shouldn't be there

**Review by silchyn — CHANGES_REQUESTED** [review]

File: src/styles/block/container.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
+.container {
+  @include centered-main-container;
+}
+
+.calendar {
```
Comment (naziamoff) [inline src/styles/block/container.scss]: yeah, does not seem to be a part of container

File: src/styles/utils/_mixins.scss [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+@mixin set-days-container-properties {
+  width: $container-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+}
+
+@mixin centered-main-container {
```
Comment (naziamoff) [inline src/styles/utils/_mixins.scss]: let's think of more descriptive naming. I should understand what is it about by its name. Properties are pretty random. They should be connected by some common thing

**Review by naziamoff — CHANGES_REQUESTED** [review]
please move your blocks to corresponding folders and fix naming

File: src/styles/utils/_reset.scss [hunk 7]
```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
```
Comment (silchyn) [inline src/styles/utils/_reset.scss:2]: does not look like resetting css

**Review by silchyn — APPROVED** [review]

**Review by silchyn — APPROVED** [review]

**Pull request closed without merging**